Update write_abstract_lef in README (#2417)
The-OpenROAD-Project/OpenROAD: src/README.md

``` tcl

> write_abstract_lef [-bloat_factor bloat_factor] \
>                    [-bloat_occupied_layers] \

Reviewer: ```suggestion
write_abstract_lef (-bloat_factor bloat_factor|-bloat_occupied_layers) \
```

``` tcl

> read reg1.db

Reviewer: ```suggestion
```

> write_abstract_lef reg1_abstract.lef \
> 		   [-bloat_factor shape_pitch_bloat_factor ] \
> 		   [-bloat_occupied_layers ] 	

Reviewer: ```suggestion
write_abstract_lef [-bloat_factor bloat_factor] \
                   [-bloat_occupied_layers] \
		   filename
```

```
### Options

> | Switch Name | Description |
> | ----- | ----- |
> | `-bloat_factor` | Specifies the bloat factor of the shapes in order for merging obstructions |
> | `-bloat_occupied_layers` | Specifies that the obstruction will include the entire layer |

Reviewer: ```suggestion
| Switch Name | Description |
| ----- | ----- |
| `-bloat_factor` | Specifies the bloat factor used when bloating then merging shapes into LEF obstructions. The factor is measured in # of default metal pitches for the respective layer. A factor of `0` will result in detailed LEF obstructions |
| `-bloat_occupied_layers` | Generates cover obstructions (obstructions over the entire layer) for each layer where shapes are present |
```

### Examples
```

> # Increase the shapes in 3 pitches (for every layer)
> write_abstract_lef reg1_abstract.lef -bloat factor 3
> 
> # The obstructions will contain the entire layer of the nets
> write_abstract_lef reg1_abstract.lef -bloat_occupied_layers

Reviewer: ```suggestion
read reg1.db

# Bloat metal shapes by 3 pitches (respectively for every layer) and then merge
write_abstract_lef -bloat factor 3 reg1_abstract.lef

# Produce cover obstructions for each layer with shapes present
write_abstract_lef -bloat_occupied_layers reg1_abstract.lef
```
